mobile/src/main/res/layout/fragment_status.xml

```diff
@@ -1,48 +1,66 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:tools="http://schemas.android.com/tools"
-              android:layout_width="match_parent"
-              android:layout_height="match_parent"
-              android:gravity="center_horizontal|center_vertical"
-              android:orientation="vertical">
-    <ImageView
-        android:id="@+id/image"
-        android:layout_width="100dp"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
-        android:adjustViewBounds="true"
-        android:gravity="center"
-        android:scaleType="fitXY"
-        android:src="@drawable/ic_signal_cellular_off_black_24dp"
-        tools:ignore="contentDescription"/>
+<ScrollView
```

Comment:
Is the ScrollView still needed? I still think we should avoid the need for scrolling in those views.

Reply:
I have no device where the view is scrollable, but IMO it's better to have a safeguard for extra small devices.

Reply:
What's the smallest screen size we (want to) support? Since our min API level is ICS I'd say 4 or 4.3 inch?
(I want to double check with emulator)

Reply:
Even if it's ok in English, it might be too much text in other languages.